src/main/java/at/hannibal2/skyhanni/utils/NEUItems.kt

```diff
@@ -88,18 +87,20 @@ object NEUItems {
     }
 
     fun readAllNeuItems(): Map<String, NEUInternalName> {
-        allInternalNames.clear()
         val map = mutableMapOf<String, NEUInternalName>()
         for (rawInternalName in allNeuRepoItems().keys) {
             val name = manager.createItem(rawInternalName).displayName.removeColor().lowercase()
             val internalName = rawInternalName.asInternalName()
             map[name] = internalName
-            allInternalNames.add(internalName)
         }
         return map
     }
 
-    fun getInternalNameOrNull(itemName: String): NEUInternalName? {
+    @Deprecated("moved", ReplaceWith("NEUInternalName.fromItemNameOrNull(itemName)"))
+    fun getInternalNameOrNull(itemName: String): NEUInternalName? = NEUInternalName.fromItemNameOrNull(itemName)
+
+    // internal function
+    fun getInternalNameOrNull0(itemName: String): NEUInternalName? {
```

Comment:
Can be really internal not just a comment
```suggestion
    internal fun getInternalNameOrNull0(itemName: String): NEUInternalName? {
```

Reply:
fixed